tasks/main.py

```diff
@@ -37,6 +37,32 @@ def replace_private_values(text, private_values,
     return text
 
 
+def find_custom_error_message(err_string):
+    '''
+    Returns a custom error message based on the current
+    contents of `err_string`, if one exists.
+
+    Otherwise, just return the supplied `err_string`.
+
+    >>> msg = find_custom_error_message('boop InvalidAccessKeyId')
+    >>> 'S3 keys were rotated' in msg
+    True
+
+    >>> find_custom_error_message('boop')
+    'boop'
+    '''
+    if "InvalidAccessKeyId" in err_string:
```

Comment:
If we end up having additional custom error messages, we could instead have a `dict` of keys and associated messages (stored in another file) to make this function less verbose, but since there is only one for now I think this is sufficient.

Reply:
:+1: